cli/update.go

```diff
@@ -55,7 +55,7 @@ func (uc *UpdateCommand) addFlags() {
 	flagSet.StringSliceVarP(&uc.labels, "label", "l", nil, "Update labels for container")
 	flagSet.StringVar(&uc.restartPolicy, "restart", "", "Restart policy to apply when container exits")
 	flagSet.StringSliceVar(&uc.diskQuota, "disk-quota", nil, "Update disk quota for container(/=10g)")
-	flagSet.StringSliceVar(&uc.specAnnotation, "annotation", nil, "Update annotation for runtime spec")
+	flagSet.StringArrayVar(&uc.specAnnotation, "annotation", nil, "Update annotation for runtime spec")
```

Comment:
Could you add more detailed description of `specAnnotation` in swagger:

```
      SpecAnnotation:
        description: "annotations send to runtime spec."
        type: "object"
        additionalProperties:
          type: "string"
```

Actually, users can hardly understand the actual meaning of it. Can you add some example for it in swagger.yml? @wangforthinker